src/structure/pure/Line.js

```diff
@@ -15,31 +15,47 @@
  * You should have received a copy of the GNU General Public License along
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA
- *
- * @module Line 
- * @title  CometVisu Line 
  */
 
 
 /**
+ * A line tag is used to display a horizontal line in the browser to allow for a grouped and thereby tidy display of
+ * elements on one page. To specify the width of the line an optional &lt;layout/&gt;-child can be added.
+ *
+ * @widget_example <meta selector="hr">
+ *   <caption>A line tag which uses 50% of the screen size (6 of 12 available columns)</caption>
+ * </meta>
+ * <line><layout colspan="6"/></line>
+ *
+ * @module structure/pure/Line
+ * @requires structure/pure
  * @author Christian Mayer
- * @since 2012
+ * @since 0.8.0 (2012)
```

Comment:
Hm, should this be the date / version of the file or of the widget?

The widget was here sice the beginning (cf. https://sourceforge.net/p/openautomation/code/HEAD/tree/CometVisu/branches/release_0.5.1/visu/visu_config.xsd ), the file was most likely created during the widget separation into a file per widget